src/leap/mail/imap/mailbox.py

```diff
@@ -462,7 +462,8 @@ def fetch(self, messages_asked, uid):
         # Change this to the flag that twisted expects when we
         # switch to content-hash based index + local UID table.
 
-        is_sequence = True if uid == 0 else False
+        #is_sequence = True if uid == 0 else False
+        is_sequence = False # lie to me so that my client works
```

Comment:
I can add a better explanation, sure, this is a debug-hack. The client I'm referring to here is the twisted.mail.imap.client, that sends sequence numbers instead of absolute uids, and adding full support for that is going to take a while.